### PR TITLE
SPE-345 finalize and migrate dex.trades to new structure

### PR DESCRIPTION
**Pull request by Hosuke**
In this PR:
- rename original dex.trades to dex.trades_legacy
- rename dex.trades_beta to dex.trades
- clean up dependencies on dex.trades_beta

### CODE REVIEW

File: models/balancer/arbitrum/balancer_v2_arbitrum_trades_beta.sql
Comment (jeff-dude) [inline models/balancer/arbitrum/balancer_v2_arbitrum_trades_beta.sql:1]: plz take note that we need to move all these balancer views off `_beta` next and remove that in the name / drop old views

we can do in follow-up PR so it's clean, tag balancer team, quick test of the views built, etc.

**Comment by jeff-dude** [comment]
i haven't forgotten about this, just need to prioritize a few other items. soon!

**Review by jeff-dude — APPROVED** [review]

**Comment by jeff-dude** [comment]
marking as `blocked`, since i need to prep some things prior to release
- internal docs
- communication plan

**Review by jeff-dude — APPROVED** [review]
all good now 🤝 
will prep to merge monday morning EST

File: models/balancer/arbitrum/balancer_v2_arbitrum_trades_beta.sql
Comment (Hosuke) [reply models/balancer/arbitrum/balancer_v2_arbitrum_trades_beta.sql:1]: Follow-up PR #5625

**Pull request merged**